### DISCUSSION TITLE
Feat: Allow for fish-like syntax highlighting in CW term

### DISCUSSION THREAD

**Discussion by fzhang13** [aws/codewhisperer-command-line-discussions#70] [Feature request]
### Checks

- [X] I could not find an existing setting that solved my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my feature request

### Description:
> Currently there isn't a way to enable syntax highlighting when using CW CLI. would be nice to have

**Comment by brendanfalk** [comment]
Great idea! There is a plugin called https://github.com/zdharma-continuum/fast-syntax-highlighting that we recommend you try. Will keep this open in the meantime
